test/operations/operations.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/kubevirt/cluster-network-addons-operator/pkg/names"
 )
 
+const (
+	OPERATOR_NAMESPACE = "cluster-network-addons-operator"
```

Comment:
The operator-sdk provide you already with the nameapace

Reply:
HI @qinqon . Thanks for the notes.  Not sure I get you here, How do I get the namespace?

Reply:
You can get it from pkg/components/components.go. That's what we do in test main.